ibis/expr/types/structs.py

```diff
@@ -85,7 +86,7 @@ def struct(
 
 
 @public
-class StructValue(Value):
+class StructValue(Value, IbisMapping):
```

Comment:
I'd really like to avoid the subclass if we can.

Is it sufficient to implement the necessary methods?

Reply:
See my comment on the PR, and the updated version of the code.